biodatasets/tmvar/tmvar.py

```diff
@@ -0,0 +1,327 @@
+                features = datasets.Features(
+                    {
+                        "pmid": datasets.Value("string"),
+                        "passages": [
+                            {
+                                "type": datasets.Value("string"),
+                                "text": datasets.Sequence(datasets.Value("string")),
+                                "offsets": datasets.Sequence([datasets.Value("int32")]),
+                            }
+                        ],
+                        "entities": [
+                            {
+                                "text": datasets.Sequence(datasets.Value("string")),
+                                "offsets": datasets.Sequence([datasets.Value("int32")]),
+                                "concept_id": datasets.Value("string"),
+                                "semantic_type_id": datasets.Sequence(
+                                    datasets.Value("string")
+                                ),
+                            }
+                        ],
+                    }
+                )
+            elif version == "v2":
+                features = datasets.Features(
+                    {
+                        "pmid": datasets.Value("string"),
+                        "passages": [
+                            {
+                                "type": datasets.Value("string"),
+                                "text": datasets.Sequence(datasets.Value("string")),
+                                "offsets": datasets.Sequence([datasets.Value("int32")]),
+                            }
+                        ],
+                        "entities": [
+                            {
+                                "text": datasets.Sequence(datasets.Value("string")),
+                                "offsets": datasets.Sequence([datasets.Value("int32")]),
+                                "concept_id": datasets.Value("string"),
+                                "semantic_type_id": datasets.Sequence(
+                                    datasets.Value("string")
+                                ),
+                                "rsid": datasets.Value("string"),
+                            }
+                        ],
+                    }
+                )
+        elif self.config.schema == "bigbio_kb":
+            features = schemas.kb_features
+
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager) -> List[datasets.SplitGenerator]:
+        """Returns SplitGenerators."""
+        version = self.config.name.split("_")[1]
+        # Hacky fix for inconsitent data format in tmvar
+        # v1 is a zip of train and test files
+        # v2 is a single .txt file with only the test data
+        urls = _URLS[_DATASETNAME][version]
```

Comment:
not a big deal, but this is singular at this point right? (i.e. it is one url)

Reply:
Yes, there's a single URL for each version

Reply:
```suggestion
        url = _URLS[_DATASETNAME][version]
```

Reply:
> 

This change has been made